Frameworks/CoreGraphics/CGBitmapContext.mm

```diff
@@ -84,16 +88,21 @@ CGImageAlphaInfo CGBitmapContextGetAlphaInfo(CGContextRef context) {
     }
 
     return (CGImageAlphaInfo)ret;
+#endif
+    return (CGImageAlphaInfo)0;
 }
 
 /**
  @Status Interoperable
  @Notes
 */
 size_t CGBitmapContextGetBitsPerComponent(CGContextRef context) {
+#if 0
     CGImageRef imageRef = context->Backing()->DestImage();
```

Comment:
I attempted to cover that [here](https://github.com/Microsoft/WinObjC/pull/1126#issuecomment-253062344). :)